src/styles/AddTodoForm.module.css

```diff
@@ -0,0 +1,15 @@
+ul {
+  width: 420px;
+  list-style: none;
+  margin: auto;
+  padding: 0;
+  color: var(--main-font-color);
+  display: flex;
+  flex-direction: column;
+  text-align: left !important ;
```

Comment:
most linters will give a warning for using `!important`, which is considered too heavy-handed and can cause unexpected bugs later on. 

Luckily this one isn't actually doing anything anymore, and the line works great with just `text-align: left`